node/src/main/scala/coop/rchain/node/configuration/hocon/model.scala

```diff
@@ -323,12 +323,11 @@ object Casper {
     import Configuration.RichConfig
 
     val casper = config.getConfig(Key)
-    val pkPath = casper.getPathOpt(keys.ValidatorPrivateKeyPath).map(Right(_))
-    val pk     = casper.getStringOpt(keys.ValidatorPrivateKey).map(Left(_))
+    val pk     = casper.getStringOpt(keys.ValidatorPrivateKey)
```

Comment:
I know this is nit picking, but I feel like it's always better to call the private key "sk" and public key "pk"

Reply:
If _pk_ is alone I'm lost. :wink: